internal/pkg/cli/interfaces.go

```diff
@@ -664,6 +671,10 @@ type secretPutter interface {
 	PutSecret(in ssm.PutSecretInput) (*ssm.PutSecretOutput, error)
 }
 
+type secretDecrypter interface {
+	GetSecretValue(s string) (string, error)
+}
+
```

Comment:
I think we need only one of the `secretDecrypter` and `secretGetter` interfaces
```suggestion
type secretDecrypter interface {
	GetSecretValue(s string) (string, error)
}

```